twilight-http/src/routing.rs

```diff
@@ -1532,7 +1537,9 @@ impl<'a> Route<'a> {
                 Path::ApplicationGuildCommandId(application_id)
             }
             Self::GetCurrentAuthorizationInformation => Path::OauthMe,
-            Self::GetCurrentUserApplicationInfo => Path::OauthApplicationsMe,
```

Comment:
should `Path::OauthApplicationsMe` be removed? i don't think it's used anymore, would that be a breaking change (is `Path` public)

Reply:
Path is public, so it'll have to stay.

Reply:
WEould it make sense to mark it as deprecated or does there still exist uses for it?

Reply:
It's kinda weird situation, the functionality between the `/applications` and `/oauth2/applications` seems to overlap when fetching the current user application, and neither have been marked as deprecated by discord. From my understanding the `/oauth2` path is meant to be used for non-bot specific situations.